### PR TITLE
Volmgr, failover, failback and other fixes

### PR DESCRIPTION
**Pull request by zedi-pramodh**
This commit fixes most of the   failover issues and state changes issues I found in my testing.
This commit can go for testing to QA to start the failover testing.

I might checkin this code and provide a build to QA without approval because its Thanksgiving weekend and I do not expect people to work.

### CODE REVIEW

File: pkg/pillar/types/clustertypes.go [hunk 1]
```diff
@@ -41,6 +41,7 @@ type ENClusterAppStatus struct {
 	IsDNSet             bool      // DesignatedNodeID is set for this node
 	ScheduledOnThisNode bool      // App is running on this device
 	StatusRunning       bool      // Status of the app in "Running" state
+	StatusStopped       bool      // App is in stopped state
```
Comment (andrewd-zededa) [inline pkg/pillar/types/clustertypes.go:44]: From a quick read-through I think it looks ok but why do we need both a StatusRunning and StatusStopped bool?

**Review by andrewd-zededa — COMMENTED** [review]
I'm confused on the StatusRunning/StatusStopped bool flags.

File: pkg/pillar/types/clustertypes.go (hunk 1 above)
Comment (zedi-pramodh) [reply pkg/pillar/types/clustertypes.go:44]: I modified this commit a bit and retesting. Will amend the commit ASAP

Comment (zedi-pramodh) [reply pkg/pillar/types/clustertypes.go:44]: Closing this because I created PR https://github.com/naiming-zededa/eve/pull/392

**Pull request closed without merging**